OpenRA.Mods.Common/Traits/Sound/AnnounceOnBuild.cs

```diff
@@ -19,7 +19,7 @@ public class AnnounceOnBuild : INotifyBuildComplete
 	{
 		public void BuildingComplete(Actor self)
 		{
-			Sound.PlayVoice("Build", self, self.Owner.Country.Race);
+			self.PlayVoice("Build", self.Owner.Country.Race);
```

Comment:
This and the "Kill" below should probably also be moved into the Info class, like the "Unload" above. Doesn't have to be in this PR, though, just sayin'.